lib/scanner/index.js

```diff
@@ -116,28 +97,6 @@ class Scanner extends Emitter {
     });
   }
 
-  checkBluetooth() {
-    if (Platform.OS == 'ios') return Promise.resolve(true);
-    if (this.checking) return this.checking;
-    debug('check bluetooth');
-
-    return this.checking = PromptBluetooth.prompt()
-      .then(enabled => {
-        if (enabled) {
-          debug('bluetooth enabled');
-          this.checking = null;
-          return true;
-        }
-
-        return this.showWarning()
-          .then(retry => {
-            this.checking = null;
-            if (!retry) return false;
-            return this.checkBluetooth();
-          });
-      });
-  }
```

Comment:
Checking bluetooth is an Android only issue (IIUC), so we've moved this into Android only code, making usre the user is prompted before the scanner runs.